src/services/twilio-help/twilio-command-help.js

```diff
@@ -47,77 +125,10 @@ class TwilioCommandHelp extends CommandHelp.default {
     if (hyperLink.isSupported) {
       listOfDetails.push(chalk.bold(hyperLink.url));
     } else {
-      listOfDetails.push(chalk.bold('MORE INFO'));
       listOfDetails.push(indent(helpDoc, 2));
     }
     return listOfDetails.join('\n');
   }
-
-  // overriding to include docs()
-  generate() {
-    const cmd = this.command;
-    const flags = util.sortBy(
-      Object.entries(cmd.flags || {})
-        .filter(([, v]) => !v.hidden)
-        .map(([k, v]) => {
-          v.name = k;
-          return v;
-        }),
-      (f) => [!f.char, f.char, f.name],
-    );
-    const args = (cmd.args || []).filter((a) => !a.hidden);
-    let output = util
-      .compact([
-        this.usage(flags),
-        this.args(args),
-        this.flags(flags),
-        this.description(),
-        this.aliases(cmd.aliases),
-        this.examples(cmd.examples || cmd.example),
-        this.docs(),
-      ])
-      .join('\n\n');
-    if (this.opts.stripAnsi) output = stripAnsi(output);
-    return output;
-  }
-
-  /**
-   *   Forked and refactored from oclif default implementation.
-   *   Link: https://github.com/oclif/plugin-help/blob/master/src/command.ts#L125-L154
-   */
-  generateFlagsOutput(flags) {
-    return list.renderList(
-      flags.map((flag) => {
-        let left = flag.helpLabel;
-        if (!left) {
-          const label = [];
-          if (flag.char) label.push(`-${flag.char[0]}`);
-          if (flag.name) {
-            if (flag.type === 'boolean' && flag.allowNo) {
-              label.push(`--[no-]${flag.name.trim()}`);
-            } else {
-              label.push(`--${flag.name.trim()}`);
-            }
-          }
-          left = label.join(', ');
-        }
-        if (flag.type === 'option') {
-          let value = flag.helpValue || flag.name;
```

Comment:
[self] 
The earlier output was: **`-p, --profile=profile`**
Current output is: **`-p, --profile=<value>`** 
This line is [modified](https://github.com/oclif/core/blob/29f76feb04e067b084009946e54cd840da683932/src/help/command.ts#L269) in oclif/core to show the flag name only when the optn `showFlagNameInTitle` is set.